### ISSUE TITLE
Does not work in Zeek v5

### ISSUE THREAD

**Issue by keithjjones** [cisagov/icsnpp-genisys#16]
# 🐛 Summary #

This line doesn't work in Zeek v5.0: https://github.com/cisagov/icsnpp-genisys/blob/main/analyzer/main.zeek#L83

There should probably be a `@if` looking at the Zeek version here for the foreseeable future.

## To reproduce ##

1. Do this
1. Then this

## Expected behavior ##

What did you expect to happen that didn't?

We should be able to use this on prior versions of Zeek for a while longer.

## Any helpful log output or screenshots ##

Paste the results here:

```console

```

Add any screenshots of the problem here.

**Comment by mmguero** [comment]
Thanks!

**Comment by mmguero** [comment]
Fixed with #17 

```
 zeek --version
zeek version 5.0.10
.root d57b9a9c200c zeek-logs> zkg install --force --skiptests /mmguero-dev/icsnpp-genisys 
Installing "/mmguero-dev/icsnpp-genisys".......
Installed "/mmguero-dev/icsnpp-genisys" (main)
Loaded "/mmguero-dev/icsnpp-genisys"
.root d57b9a9c200c zeek-logs> zeek -NN local 2>&1 | grep -i genisys
    [Analyzer] GENISYS_TCP (ANALYZER_GENISYS_TCP, enabled)
    [Type] Genisys::HeaderCode
    [Type] Genisys::Direction
```

```
.root f3fb660df21e zeek-logs> zeek --version
zeek version 6.1.0
.root f3fb660df21e zeek-logs> zkg install --force /mmguero-dev/icsnpp-genisys/
Running unit tests for "/mmguero-dev/icsnpp-genisys"
Installing "/mmguero-dev/icsnpp-genisys"........
Installed "/mmguero-dev/icsnpp-genisys" (main)
Loaded "/mmguero-dev/icsnpp-genisys"
.root f3fb660df21e zeek-logs> zeek -NN local 2>&1 | grep -i genisys
    [Analyzer] GENISYS_TCP (ANALYZER_GENISYS_TCP, enabled)
    [Type] Genisys::HeaderCode
    [Type] Genisys::Direction
```
